WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
+                XS_13TeV=156.7, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_800to1000" : {
             "CrossSection" : XSValues(
-                XS_2016=32.293, XSSource_2016="AN2017_013_v17, XSDB 32.05",
-                XS_2017=26.28, XSSource_2017="XSDB",
-                XS_2018=26.32, XSSource_2018="XSDB",
+                XS_13TeV=26.25, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_1000to1400" : {
             "CrossSection" : XSValues(
-                XS_2016=9.4183, XSSource_2016="AN2017_013_v17, XSDB 9.365",
-                XS_2017=7.47, XSSource_2017="XSDB",
-                XS_2018=7.50, XSSource_2018="XSDB",
+                XS_13TeV=7.465, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_1400to1800" : {
             "CrossSection" : XSValues(
-                XS_2016=0.84265, XSSource_2016="AN2017_013_v17, XSDB 0.8398",
-                XS_2017=0.6484, XSSource_2017="XSDB",
-                XS_2018=0.6479, XSSource_2018="XSDB",
+                XS_13TeV=0.6487, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_1800to2400" : {
             "CrossSection" : XSValues(
-                XS_2016=0.114943, XSSource_2016="AN2017_013_v17, XSDB 0.1124",
-                XS_2017=0.08743, XSSource_2017="XSDB",
-                XS_2018=0.08715, XSSource_2018="XSDB",
+                XS_13TeV=0.08734, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_2400to3200" : {
             "CrossSection" : XSValues(
-                XS_2016=0.00682981, XSSource_2016="AN2017_013_v17, XSDB 0.006752",
-                XS_2017=0.005236, XSSource_2017="XSDB",
-                XS_2018=0.005242, XSSource_2018="XSDB",
+                XS_13TeV=0.005237, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_3200toInf" : {
             "CrossSection" : XSValues(
-                XS_2016=0.000165445, XSSource_2016="AN2017_013_v17, XSDB 0.0001626",
-                XS_2017=0.0001357, XSSource_2017="XSDB",
-                XS_2018=0.0001349, XSSource_2018="XSDB",
+                XS_13TeV=0.0001352, XSSource_13TeV="XSDB (unknown)",
             ),
         },
-        "DYJetsToLL_M-50_HT-100to200" : {
+        "DYJetsToLL_M-50_HT-70to100" : {
             "CrossSection" : XSValues(
-                XS_2016=147.4, XSSource_2016="XSDB (LO)",
-                XS_2017=161.1, XSSource_2017="XSDB (LO)",
-                XS_2018=160.8, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=139.9, XSSource_13TeV="GenXSecAnalyzer",
             ),
-            "kFactor" : kFactorValues(
-                kFactor_2016=1.2245, kFactorSource_2016="XSDB NNLO/LO=6077.22/4963",
-                kFactor_2017=1.1374, kFactorSource_2017="XSDB NNLO/LO=6077.22/5343",
-                kFactor_2018=1.1421, kFactorSource_2018="XSDB NNLO/LO=6077.22/5321",
-            ),
-            "Correction" : CorrValues(
```

Comment:
We'll probably want to double-check after the initial production that these correction factors are no longer needed.